web-app/packages/lib/src/modules/project/components/ProjectAccessRequestTableTemplate.vue

```diff
@@ -166,30 +178,45 @@ export default Vue.extend({
         await this.acceptProjectAccessRequest({
           data,
           itemId: request.id,
-          refetchGlobalAccessRequests: true,
           namespace: this.namespace
         })
+        await this.updatePaginationOrFetch()
       } catch (err) {
         this.$emit('accept-access-request-error', err)
       }
     },
-    changeSort(column) {
-      if (this.options.sortBy === column) {
-        this.options.descending = !this.options.descending
-      } else {
-        this.options.sortBy = column
-        this.options.descending = false
-      }
-    },
     expired(expire) {
       return Date.parse(expire) < Date.now()
     },
     async cancelRequest(request) {
       await this.cancelProjectAccessRequest({
         itemId: request.id,
-        refetchGlobalAccessRequests: true,
         namespace: this.namespace
       })
+      await this.updatePaginationOrFetch()
+    },
+    async fetchItems() {
+      this.loading = true
+      try {
+        const payload: GetProjectAccessRequestsPayload = {
+          namespace: this.namespace,
+          params: {
+            page: this.options.page,
+            per_page: this.options.itemsPerPage,
+            order_params:
+              this.options.sortBy[0] &&
```

Comment:
should not we support multi-column sorting?

Reply:
This interface is from vuetify data-table options.